.travis.yml

```diff
@@ -17,6 +17,7 @@ install:
   - pip install -q flake8 coveralls
   - pip install -q https://github.com/qiime2/q2lint/archive/master.zip
   - make install
+  - conda install -c conda-forge umap-learn
```

Comment:
is that the best way to add the conda package to the installation.

Reply:
The `umap-learn` dependency should be added to the package recipe, https://github.com/qiime2/q2-diversity/blob/cb84424cd3707432e63ed400a1b90345dc8d2e22/ci/recipe/meta.yaml#L21, and removed from this `.travis.yml` config.

Reply:
okay, thank you for the hint!